.ci/load/README.md

```diff
@@ -0,0 +1,61 @@
+# APM Java Agent Load Geneator
+
+This directory contains configuration files for load-testing the APM Java Agent.
+
+## Load Tests
+
+Load tests are run using two bare-metal workers. One worker is dedicated to the role of load generation, which other worker is dedicated to the role of hosting the application under test, which is instrumented by the APM Java Agent.
+
+Load generation is performed using [Locust](https://locust.io/) and the application which is instrumented by the APM Java Agent and placed under load is [Spring Petclinic](https://projects.spring.io/spring-petclinic/).
```

Comment:
Is it possible to view the Locust results? If so, how?

Reply:
Yes. It's not super-clean because you have to basically just navigate to the Blue Ocean UI in Jenkins and then scroll to the end of the console log for the load generation step, but the output is there. It will look like this:

<img width="1523" alt="loc" src="https://user-images.githubusercontent.com/111616/97887538-f7981e80-1d21-11eb-9fb1-a08ce025c2c1.png">

I would like to find a way to present this as an artifact of the test results so it can be easily downloaded but I just made this a future TODO. Let me know if you'd like me to prioritize improving this experience because it's definitely possible.

Reply:
sounds like the right tradeoff for an MVP